### ISSUE TITLE
Could not load `container-ima.ko`

### ISSUE THREAD

**Issue by lkatalin** [avery-blanchard/container-ima#10]
```
[ebpf@fedora container-ima]$ ls
container_ima_api.h     container_ima_fs.h    container_ima.ko     container_ima.mod.o  modules.order   probe        README.md
container_ima.c         container_ima.h       container_ima.mod    container_ima.o      Module.symvers  probe.bpf.c
container_ima_crypto.h  container_ima_init.h  container_ima.mod.c  Makefile             output          probe.c

[ebpf@fedora container-ima]$ sudo insmod container-ima.ko 
insmod: ERROR: could not load module container-ima.ko: No such file or directory
```

**Comment by lkatalin** [comment]
This may have been a perms issue, after `chmod 777 container_ima.ko` I get

```
[ebpf@fedora container-ima]$ sudo insmod container_ima.ko
insmod: ERROR: could not insert module container_ima.ko: Unknown symbol in module

```

**Comment by lkatalin** [comment]
Update: it was a typo

**Comment by lkatalin** [comment]
This is fixed in #11
